docs/code-quality/includes/fix-binaryformatter.md

```diff
@@ -0,0 +1,2 @@
+- Use a secure serializer instead, and **don't allow an attacker to specify an arbitrary type to deserialize**. For more information see the [Preferred alternatives](/dotnet/standard/serialization/binaryformatter-security-guide#preferred-alternatives).
+- Make the serialized data tamper-proof. After serialization, cryptographically sign the serialized data. Before deserialization, validate the cryptographic signature. Protect the cryptographic key from being disclosed and design for key rotations.
```

Comment:
@GrabYourPitchforks I removed the following, should I add it back?

Restrict deserialized types. Implement a custom System.Runtime.Serialization.SerializationBinder. Before deserializing with BinaryFormatter, set the Binder property to an instance of your custom SerializationBinder. In the overridden BindToType method, if the type is unexpected, throw an exception to stop deserialization.
Ensure that all code paths have the Binder property set.